tests/integration/conftest.py

```diff
@@ -101,6 +123,7 @@ async def app(
     ops_test: OpsTest,
     app_name: str,
     content_cache_image: str,
+    nginx_exporter_image: str,
```

Comment:
I realise it's more verbose, but I think we should name this nginx_prometheus_exporter_image to be consistent with the image name.